pkgs/build-support/fetchgit/nix-prefetch-git

```diff
@@ -76,7 +80,7 @@ for arg; do
     else
         case $argfun in
             set_*)
-                var=$(echo $argfun | sed 's,^set_,,')
+                var=${argfun/#set_/}
```

Comment:
I think you should remove the `#` symbol.  Otherwise this does not behave as expected.